src/bgnbd_expectation.cpp

```diff
@@ -0,0 +1,82 @@
+#include <RcppArmadillo.h>
+#include <math.h>
+#include "bgnbd_expectation.h"
+#include "clv_vectorized.h"
+
+//' @name bgnbd_expectation
+//' @title BG/NBD: Unconditional Expectation
+//'
+//' @description Computes the expected number of repeat transactions in the interval (0, vT_i]
+//' for a randomly selected customer, where 0 is defined as the point when the customer came alive.
+//'
+//' @template template_params_bgnbd
+//' @param vT_i Number of periods since the customer came alive
+//' @template template_params_rcppcovmatrix
+//' @template template_params_rcppvcovparams
+//'
+//' @templateVar name_params_cov_life vCovParams_life
+//' @templateVar name_params_cov_trans vCovParams_trans
+//' @template template_details_rcppcovmatrix
+//'
+//' @template template_references_bgnbd
```

Comment:
return value needs to be documented as well. unfortunately also not done in ggomnd expectation... make template, re-use across expectation functions